audbcards/core/dataset.py

```diff
@@ -25,12 +43,87 @@ class Dataset:
         cache_root: cache folder
 
     """
+
+    def __new__(
+        cls,
+        name: str,
+        version: str,
+        *,
+        cache_root: str = "~/.cache",
```

Comment:
I would prefer that you can see the actual cache folder here, and the user is not afraid that `~/.cache` gets polluted by files. So I would change here to the actual location `~/.cache/audbcards`.

Reply:
set to `~/.cache/audbcards`

Reply:
You also need to change the underlying code. Now the cache is stored inside `~/.cache/audbcards/audbcards/`

Reply:
It would still be nice if it would be stored inside `~/.cache/audbcards/` instead.

Reply:
Ah sorry I forgot the underlying implementation.
This is changed in 8d8ffd0  now.

In my understanding emodb might end up in `~/.cache/audbcards/emodb/1.2.1/emodb-1.2.1.pkl` - when using the default of ` ~/.cache/audbcards/`

Reply:
Yes, it lands in `~/.cache/audbcards/emodb/1.2.1/emodb-1.2.1.pkl`.
Before the cache for `Dataset` was separated from the actual data cache as loaded by `audb`, and the file was stored under `~/.cache/audbcards/dataset/emodb/1.2.1/emodb-1.2.1.pkl`.
But I think your new solution makes more sense as it will then be easier to delete the cache associated with a single dataset, by just deleting a single folder, e.g. `~/.cache/audbcards/emodb`.

Reply:
In https://github.com/audeering/audbcards/issues/29 I have discussed that we might separate the cache folders of `audb.Dataset` and `audb.Datacard` as `~/.cache/audbcards/dataset` and `~/.cache/audbcards/datacard`.
I guess in `audb.Datacard` we might want to cache the selected audio files and maybe some PNG files.

On the other hand, we might also want to add the cache `audb.Datacard` files under e.g. `~/.cache/audbcards/emodb` so that we still have everything together in a single dataset related folder. The only downside is we then need to be careful that `audb` can still load the dataset from that folder and we don't mess up with its cache handling.

Any opinion what would be the best solution here?

Reply:
I am not sure whether I am aware of the problem:

the `audb` - cache would very likeliky be entirely separated from the `audbcards` cache in the current solution.
So this might end up in  `~/.audb` (possibly symliked to somewhere else) as the audb cache and  `~/.cache/audbcards/ `  for audbcards.

The audb files are `db.yaml`  and located in a flavor-hashed subdirectory in the cache.  
What scenarios can possibly create interferences here?

Would something like this be viable?

```
.                                                                                                                                                                                                                                                                                                                                                                                                                                           
└── 1.2.1                                                                                                                                                                                                                                                                                                                                                                                                                                   
    ├── assets                                                                                                                                                                                                                                                                                                                                                                                                                              
    │   ├── demo.wav                                                                                                                                                                                                                                                                                                                                                                                                                        
    │   ├── image001.png                                                                                                                                                                                                                                                                                                                                                                                                                    
    │   └── image002.png                                                                                                                                                                                                                                                                                                                                                                                                                    
    └── emodb-1.2.1.pkl                                                                                                                                                                                                                                                                                                                                                                                                                     
                                                                                                                                                                                                                                                                                                                                                                                                                                            
2 directories, 4 files                                                                                                                                                                                                                                                                                                                                                                                                                      
```

One in addition could have additional renamings:

- `assets` -> `./assets` or even name it `.datacard` 
- have the origin from the dataset incoded in the pickle:  `emodb-1.2.1.pkl` -> `emodb-1.2.1-datacard.pkl`

Reply:
No, you are right. The correct solution might indeed be that we use the default `audb` cache instead of setting it also to `~/.cache/audbcards/` as we do at the moment.
But I would also propose to do it in another pull request, and stay with your current solution here.